src/Components/Footer/footer.css

```diff
@@ -1,6 +1,7 @@
 .footer{
+    background-color: blueviolet;
     height: 50px;
-    margin-top: auto;
     width: 100%;
     color: brown;
+    margin-bottom: -50px;
 }
```

Comment:
footer should have position fixed
. footer {
...
position: fixed;
bottom: 0;
}